test/transforms.test.js

```diff
@@ -9,94 +9,148 @@ import {
-      default: true
-    });
+      await Migrate('@percy/sdk-test', '--skip-cli');
 
-    expect(prompts[3]).toEqual({
-      type: 'confirm',
-      name: 'doTransform',
-      message: 'How about this one?',
-      default: true
-    });
+      expect(prompts[3]).toEqual({
+        type: 'input',
+        name: 'filePaths',
+        message: 'Which files?',
+        default: 'test/**/*.test.js',
+        filter: expect.any(Function)
+      });
 
-    expect(logger.stderr).toEqual([]);
-    expect(logger.stdout).toEqual([
-      '[percy] Migration complete!\n'
-    ]);
-  });
+      expect(prompts[5]).toEqual({
+        type: 'input',
+        name: 'filePaths',
+        message: 'Which files?',
+        default: 'test/**/*.test.js',
+        filter: expect.any(Function)
+      });
 
-  it('asks for filepaths to transform when confirmed', async () => {
-    prompts = mockPrompts({
-      isSDK: true,
-      doTransform: true,
-      filePaths: q => q.filter(q.default)
-        .then(f => f.sort())
+      expect(transformed).toEqual(
+        await globby('test/**/*.test.js')
+          .then(f => f.sort())
+      );
+
+      expect(logger.stderr).toEqual([]);
+      expect(logger.stdout).toEqual([
+        '[percy] Migration complete!\n'
+      ]);
     });
+  });
 
-    await Migrate('@percy/sdk-test', '--skip-cli');
+  describe('without matching paths', () => {
+    let transformed;
 
-    expect(prompts[3]).toEqual({
-      type: 'input',
-      name: 'filePaths',
-      message: 'Which files?',
-      default: 'test/**/*.test.js',
-      filter: expect.any(Function)
-    });
+    beforeEach(() => {
+      transformed = [];
+
+      mockMigrations([{
+        name: '@percy/sdk-test',
+        version: '^2.0.0',
+        transforms: [{
+          message: 'Never match',
+          default: 'no/files/should/match.ha',
+          transform: () => {}
+        }]
+      }]);
+
+      mockPackageJSON({
+        devDependencies: {
+          '@percy/sdk-test': '^1.0.0'
+        }
+      });
 
-    expect(prompts[5]).toEqual({
-      type: 'input',
-      name: 'filePaths',
-      message: 'Which files?',
-      default: 'foo/**/*.bar.js',
-      filter: expect.any(Function)
+      mockPrompts({
+        isSDK: true,
+        doTransform: false
+      });
     });
 
-    expect(transformed).toEqual(
-      await globby('test/**/*.test.js')
-        .then(f => f.sort())
-    );
+    it('logs an error when no matching paths found', async () => {
+      mockPrompts({
+        isSDK: true,
+        doTransform: true,
+        filePaths: q => q.filter(q.default)
+          .then(f => f.sort())
+      });
 
-    expect(logger.stderr).toEqual([]);
-    expect(logger.stdout).toEqual([
-      '[percy] Migration complete!\n'
-    ]);
+      await Migrate('@percy/sdk-test', '--skip-cli');
+
+      expect(transformed).toEqual(
+        await globby('no/files/should/match.ha')
+          .then(f => f.sort())
+      );
+      expect(transformed).toEqual([]);
```

Comment:
Double expectation snuck in here. But if the transform is changed according to my suggestion, we can `expect(transformed).toBe(false)` or `expect(transform).not.toBe(true)`